### ISSUE TITLE
support dandi identifiers

### ISSUE THREAD

**Issue by satra** [dandi/dandi-cli#89]
the dandi cli should support dandi identifiers. since we know what this maps to we should be able to route it directly without going through identifiers.org. so the following should all be feasible.

```shell
dandi download DANDI:000008
dandi download https://identifiers.org/DANDI:000008
dandi download https://dandiarchive.org/dandiset/000008
```

**Comment by yarikoptic** [comment]
the API and identifiers should have been now handled with #108 . 
The first shortcut: I think we can do it (I love human friendly sugarings), but I wonder if we should consider it (non-URL) as a shortcut for `http://identifiers.org/` URLs. That would allow for supporting possible future alternative deployments.  But allowing for any such thing would then kill us a possibility to support e.g. `download` from some other location than internet, e.g. an ssh host (with hostname `DANDI`).  So we should decide on what usecases we would like to support

**Comment by satra** [comment]
@yarikoptic is the first one supported: `dandi download DANDI:000008`?

**Comment by yarikoptic** [comment]
will be fixed by ac89dcfb4142ae78bc846df90477b9ec07b67852 (if not rewritten) which adds handling of `DANDI:` as part of the #134
